src/EnergyPlus/WindowManager.cc

```diff
@@ -959,10 +946,6 @@ namespace WindowManager {
                 for (IGlass = 1; IGlass <= NGlass; ++IGlass) {
                     // Override 10 degree increment for incident angle for a construction with a layer = SpectralAndAngle
                     LayPtr = Construct(ConstrNum).LayerPoint(LayerNum(IGlass));
-                    if (Material(LayPtr).GlassSpectralAndAngle) {
-                        Phi = TableLookup(PerfCurve(Material(LayPtr).GlassSpecAngTransDataPtr).TableIndex).X1Var(IPhi);
-                        break;
-                    }
```

Comment:
When the section is removed, this loop of IGlass become useless. How do you justify it?

Reply:
The loop over IGlass is still used to set `wlt`, `tPhi`, `rfPhi`, and `rbPhi`.